cfme/fixtures/pytest_selenium.py

```diff
@@ -305,6 +322,14 @@ def wait_for_ajax():
     """
 
     _thread_local.ajax_log_msg = ''
+    orig_events = get_events()
+
+    def _events_not_finished():
+        events = get_events()
+        not_finished = False
+        not_finished |= orig_events["maxTimeoutId"] >= events['minTimeoutId'] > 0
```

Comment:
More comments here please. Why do we check if `events['minTimeoutId'] > 0` here? This expression evaluates False in case maxTimeoutID is e.g. 5 and minTimeoutId is 0... `5 >= 0 > 0` means `not_finished = False` which means `finished = True`... is this correct?